sonar-python-plugin/src/main/java/org/sonar/plugins/python/PythonSquidSensor.java

```diff
@@ -93,9 +100,48 @@ public void analyse(Project project, SensorContext context) {
     scanner.scanFiles(Lists.newArrayList(fileSystem.files(p.and(p.hasType(InputFile.Type.MAIN), p.hasLanguage(Python.KEY)))));
 
     Collection<SourceCode> squidSourceFiles = scanner.getIndex().search(new QueryByType(SourceFile.class));
+    savePreciseIssues(
+      visitors
+        .stream()
+        .filter(v -> v instanceof PythonCheck)
+        .map(v -> (PythonCheck) v)
+        .collect(Collectors.toList()),
+      context);
     save(squidSourceFiles);
   }
 
+  // fixme (Lena): unit test it with new Sensor API
+  private void savePreciseIssues(List<PythonCheck> pythonChecks, SensorContext context) {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "SensorContext"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)